share/doc/xml/dax.xml

```diff
@@ -87,6 +87,19 @@ unix% ds9 -analysis $ASCDS_INSTALL/contrib/config/ciao.ds9 ...
 </DESC>
 
 
+<ADESC title="Changes in the 4.12.2 (March 2020) release">
+
+  <PARA>
```

Comment:
I think this message (and the one in the changelog) reads like there is still the chance of there being a hang. Perhaps change the second sentence to

```
This stops the task from hanging if users have changed the default region format to 'ciao' format.
```

Or I mis-understand things again.

Reply:
Is the latest better?

[minor fight w/ git trying to resolve merge in Changes file ]